src/style.css

```
@@ -1 +1,37 @@
 /* add styles here */
+html {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  list-style-type: none;
+  background-image: url('images/star.svg');
+  background-size: cover;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url('images/star-active.svg');
}
```
combine it all with a comma